src/style.css

```diff
@@ -0,0 +1,61 @@
+* {
+  margin: 0;
+}
+
+.star {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(./images/star.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: 50%;
+}
+
+.star > .stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 > .stars__star:nth-of-type(1) {
+  background: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--2 > .stars__star:nth-of-type(1),
+.stars--2 > .stars__star:nth-of-type(2) {
+  background: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--3 > .stars__star:nth-of-type(1),
+.stars--3 > .stars__star:nth-of-type(2),
+.stars--3 > .stars__star:nth-of-type(3) {
+  background: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--4 > .stars__star:nth-of-type(1),
+.stars--4 > .stars__star:nth-of-type(2),
+.stars--4 > .stars__star:nth-of-type(3),
+.stars--4 > .stars__star:nth-of-type(4) {
+  background: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--5 > .stars__star:nth-of-type(1),
+.stars--5 > .stars__star:nth-of-type(2),
+.stars--5 > .stars__star:nth-of-type(3),
+.stars--5 > .stars__star:nth-of-type(4),
+.stars--5 > .stars__star:nth-of-type(5) {
+  background: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
```

Comment:
Actually this code can be shortened to 7 lines. Let's use correct condition in `nth-of-type` and combine rules with same properties to one